doc/openapi-openshift-dev-spaces.md

```diff
@@ -30,7 +30,7 @@ For this tutorial, we'll:
 To install `kuadrantctl` in your Dev Spaces workspace, execute the following command:
 
 ```bash
-curl -sL "https://github.com/kuadrant/kuadrantctl/releases/download/v0.2.0/kuadrantctl-v0.2.0-linux-amd64.tar.gz" | tar xz -C /home/user/.local/bin
+curl -sL "https://github.com/kuadrant/kuadrantctl/releases/download/v0.2.2/kuadrantctl-v0.2.2-linux-amd64.tar.gz" | tar xz -C /home/user/.local/bin
```

Comment:
yeah, will need to keep this up to date for the time being. might look at renaming the archives so they have a consistent name, so that, for instance, something like this would always work and be current:

https://github.com/kuadrant/kuadrantctl/releases/download/latest/kuadrantctl-linux-amd64.tar.gz